doc/manual/noteworthy-differences.rst

```diff
@@ -82,38 +82,102 @@ differences that may trip up Julia users accustomed to MATLAB:
 Noteworthy differences from R
 -----------------------------
 
-One of Julia's goals is to provide an effective language for data analysis and statistical programming. For users coming to Julia from R, these are some noteworthy differences:
+One of Julia's goals is to provide an effective language for data analysis
+and statistical programming. For users coming to Julia from R, these are some
+noteworthy differences:
 
-- Julia uses ``=`` for assignment. Julia does not provide any operator like ``<-`` or ``<<-``.
-- Julia constructs vectors using brackets. Julia's ``[1, 2, 3]`` is the equivalent of R's ``c(1, 2, 3)``.
-- Julia's matrix operations are more like traditional mathematical notation than R's. If ``A`` and ``B`` are matrices, then ``A * B`` defines a matrix multiplication in Julia equivalent to R's ``A %*% B``. In R, this same notation would perform an elementwise Hadamard product. To get the elementwise multiplication operation, you need to write ``A .* B`` in Julia.
-- Julia performs matrix transposition using the ``'`` operator. Julia's ``A'`` is therefore equivalent to R's ``t(A)``.
-- Julia does not require parentheses when writing ``if`` statements or ``for`` loops: use ``for i in [1, 2, 3]`` instead of ``for (i in c(1, 2, 3))`` and ``if i == 1`` instead of ``if (i == 1)``.
-- Julia does not treat the numbers ``0`` and ``1`` as Booleans. You cannot write ``if (1)`` in Julia, because ``if`` statements accept only booleans. Instead, you can write ``if true``.
-- Julia does not provide ``nrow`` and ``ncol``. Instead, use ``size(M, 1)`` for ``nrow(M)`` and ``size(M, 2)`` for ``ncol(M)``.
-- Julia's SVD is not thinned by default, unlike R. To get results like R's, you will often want to call ``svd(X, true)`` on a matrix ``X``.
-- Julia is careful to distinguish scalars, vectors and matrices. In R, ``1`` and ``c(1)`` are the same. In Julia, they can not be used interchangeably. One potentially confusing result of this is that ``x' * y`` for vectors ``x`` and ``y`` is a 1-element vector, not a scalar. To get a scalar, use ``dot(x, y)``.
+- Julia's single quotes are for characters, not strings.
+- Strings are indexable in Julia. In R, they need to be converted.
+- Strings can be created with triple quotes """ ... """ like Python.
+- Varargs in Julia are named like f(a,b,x...), unlike R which ... needs to
+  be casted to use.
+- Modulus in Julia is %, not %%.
+- Most data structures don't take a logical (boolean) vector to select off
```

Comment:
Yes, it's logical indexing -- but where you recycle `c(TRUE, FALSE)` to become `c(TRUE, FALSE, TRUE, FALSE)`, then do real logical indexing.